packages/hash/frontend/src/pages/[account-slug]/entities/[entity-uuid].page/shared/use-create-new-entity-and-redirect/get-default-value-of-property-type.ts

```diff
@@ -0,0 +1,38 @@
+import { PropertyType } from "@blockprotocol/type-system-web";
+import { Subgraph } from "@hashintel/hash-subgraph";
+import {
+  isPropertyValueArray,
+  isPropertyValueNested,
+} from "../../../../../../lib/typeguards";
+import { generateDefaultProperties } from "./generate-default-properties";
+
+/**
+ * @todo this will be deleted when https://app.asana.com/0/1203312852763953/1203433085114587/f (internal) is implemented
+ */
+export const getDefaultValueOfPropertyType = (
+  propertyType: PropertyType,
+  subgraph: Subgraph,
+) => {
+  /**
+   * if there are multiple expected types, those are not expected to be arrays or nested properties.
```

Comment:
I don't think I understand this comment. Why is this the case?

Reply:
If a property type has multiple expected types, it means it's  "boolean OR number OR string" for now (based on what we support now). So empty string works for boolean / number / string. Let's not worry about this too much, because I'll remove this code pretty soon (as I mentioned in the @todo comments here)

Reply:
Okay – does @luisbettencourt know this, as he's introducing array property types now – https://github.com/hashintel/hash/pull/1513

Reply:
No idea, but I don't think part of the code is related with how entity type editor works. Not sure about what is your concern here.